api/python/README.md

```diff
@@ -25,11 +25,23 @@ make install-dev
 make unit-tests
 ```
 
-## Release
+## Release Process
 
-The [release.yml](../../.github/workflows/release.yml) GHA should handle the release process for the pypi package.
-However if you need to manually release the package, you can use the following commands:
+A new version of the PyPi package will be released for the following reasons (not an exhaustive list):
+
+- A release will be created when functionality is added, changed, or removed.
+- A new version of the API will likely be needed if the shape of the ontology asset release changes.
+- A new version will be created when cellxgene schema change to reflect its compatibility with the new version of the CellxGene schema.
```

Comment:
add that this will occur whenever any of the ontology versions associated with a schema release are changed